spectral_cube/spectral_cube.py

```diff
@@ -1846,6 +1846,67 @@ def hdu(self):
         hdu = fits.PrimaryHDU(self.filled_data[:].value, header=self.header)
         return hdu
 
+    def plot_channel_maps(self, output_file, nx, ny, channels, levels,
+                          figsize=(20,20), decimals=3, zoom=1):
+        """
+        Make channel maps from a spectral cube
+
+        Parameters
+        ----------
+        input_file : str
+            Name of the input spectral cube
+        output_file : str
+            Name of the matplotlib plot
+        nx, ny : int
+            Number of sub-plots in the x and y direction
+        channels : list
+            List of channels to show
+        levels : list
+            List of levels to show
+        figsize : tuple, optional
+            Figure size for matplotlib
+        decimals : int, optional
+            Number of decimal places to show in spectral value
+        zoom : int, optional
+            How much to zoom in. In future versions of this function, the
+            pointing center will be customizable.
+        """
+
+        import matplotlib.pyplot as plt
+
+        if len(channels) != nx * ny:
+            raise ValueError("Number of channels should be equal to nx * ny")
+
+        # Read in spectral cube and get spectral axis
+        spectral_axis = self.spectral_axis
+
+        fig = plt.figure(figsize=figsize)
```

Comment:
should allow the figure object to be passed in, or default to using `plt.gcf()`.  But it is important to specify figsize... not sure about the best API here.